pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot-extended-configuration-options.md

```diff
@@ -0,0 +1,190 @@
+## Manual constants configuration
+
+Reading this means that you're ready for a bit of manual labour.
+If for some reason you've missed the automatic server setup section, you can read about it in the bot contributing guide [here](../bot.md#envserver)
+
+To configure the bot manually, you will **only** need to set inside the `.env.server` file the values for the channels, roles, categories, etc.
+that are used by the component you are developing.
+
+For example, if we're testing a feature that only needs the `announcements` channel:
+
+`constants.py`
+
+```py
+
+class EnvConfig:
+    # Defines from where & how Pydantic will be looking for env variables
+    ...
+
+class _Channels(EnvConfig):
+
+    EnvConfig.Config.env_prefix = "channels_"
+
+    announcements = 1079790565794779156
+    changelog = 1077877318564991006
+
+# Instantiate the class & load the configuration
+Channels = _Channels()
+```
+
+`.env.server` file:
+
+```text
+# .env.server
+
+channels_announcements=1077875228002234398
+```
+
+When you launch your bot, `pydantic` will load up the server constants from the `.env.server` file if they exist.
+
+Each constants class will define its own prefix, which will make `pydantic` look for variables that will look like `{{env_prefix}}{{attribute_name}}` in the environment files
+
+In our example, this will imply that pydantic will look for both `channels_announcements` and `channels_changelog` in the `.env.server` file.
+
+As you can see here, only `channels_announcements` has been defined in the `.env.server` file since it's the only one needed, which will tell `pydantic`
+to use the value **1077875228002234398** for the `announcements` attribute instead of the default **1079790565794779156**, and use the default value for the `changelog` attribute
+
+```python
+>>> Channels.announcements
+1077875228002234398
+>>> Channels.changelong
+1077877318564991006
+```
+
+See [here](../obtaining-discord-ids) for help with obtaining Discord IDs.
+
+If you wish to set all values in your `env.server` for your testing server, you need to set **all** the ones prefixed with:
+
+* `guild_`
+* `categories_`
+* `channels_`
+* `roles_`
+* `webhooks_`
+* `emojis_`
+
+## Working with the help forum
+
+**Note**: This is only required when you're not configuring the bot [automatically](#automatic-configuration)
+
+If you will be working on a feature that includes the python help forum, you will need to use `Forum Channels`.
+
+Forum channels cannot be included in a template, which is why this needs to be done by hand for the time being.
+
+To activate forum channels, your Discord server needs to have the community feature.
+If that's not the case already, here are the steps required to do it:
+1. Go to server settings
+2. Scroll down to the `COMMUNITY` section and click on `Enable Community`
+3. Click on `Get Started` and fill out the necessary info
+
+Once the previous steps are done, all that is left is to:
+1. Create a new channel
+2. Choose the `Forum` type
```

Comment:
This currently renders like this

![image](https://user-images.githubusercontent.com/48383734/233777841-0bfc695d-b176-4188-8436-f0d16842d19d.png)

Reply:
Fixed in a forcepush https://github.com/python-discord/site/compare/5c0e04fd156844c599c4a8b76e12d7223550f65e..a5b7c5073531036d7c03502c6d6aa5bdcd506b44